website/docs/add-secure-apps/applications/manage_apps.md

```diff
@@ -35,6 +39,18 @@ When multiple policies/groups/users are attached, you can configure the _Policy
 - Require users to pass all bindings/be member of all groups (ALL), or
 - Require users to pass either binding/be member of either group (ANY)
 
+### Application Entitlements
+
+Another method to control which users or groups can access an application is to create an Application Entitlement (which defines the specific application(s)), and then bind that to specific groups or users.
+
+1. To create an Application Entitlement open the Admin interface and navigate to **Applications -> Applications**.
+2. Click the **Application entitlements** tab, and then click **Create entitlement**. Provide a name for the entitlement, enter any optional **Attributes** , and then click **Create**.
```

Comment:
I don't see this feature on my latest build of `main`.  Does it need to be enabled somewhere?

Do you navigate to a new tab on the ApplicationsList page, or do you navigate to a specific Application and bind it?  If the latter, there's a step missing between 1 and 2: "Choose an application."